examples/java_ol_postgresql_odo/README.md

```diff
@@ -0,0 +1,510 @@
+  name: my-demo-db
+  ccpimage: crunchy-postgres-ha
+  user: testuser
+  standby: false
+  ccpimagetag: ubi7-12.3-4.4.0
+  podAntiAffinity:
+    default: preferred
+    pgBackRest: preferred
+    pgBouncer: preferred
+  backrestS3Region: ''
+  policies: ''
+  backrestS3Endpoint: ''
+  syncReplication: null
+  backrestRepoPath: ''
+  tablespaceMounts: {}
+  status: completed
+  pgBouncer:
+    limits: {}
+    replicas: 0
+    resources:
+      memory: 24Mi
+  backrestLimits: {}
+  customconfig: ''
+  pgoimageprefix: registry.connect.redhat.com/crunchydata
+  shutdown: false
+  limits: {}
+  WALStorage:
+    accessmode: ''
+    matchLabels: ''
+    name: ''
+    size: ''
+    storageclass: ''
+    storagetype: ''
+    supplementalgroups: ''
+  PrimaryStorage:
+    accessmode: ReadWriteOnce
+    matchLabels: ''
+    name: my-demo-db
+    size: 5Gi
+    storageclass: rook-ceph-cephfs-internal
+    storagetype: dynamic
+    supplementalgroups: ''
+  ArchiveStorage:
+    accessmode: ''
+    matchLabels: ''
+    name: ''
+    size: ''
+    storageclass: ''
+    storagetype: ''
+    supplementalgroups: ''
+  tls:
+    caSecret: ''
+    replicationTLSSecret: ''
+    tlsSecret: ''
+  namespace: readmeapp
+  database: my-demo-db
+  replicas: '0'
+  pgDataSource:
+    restoreFrom: ''
+    restoreOpts: ''
+  primarysecretname: my-demo-db-primaryuser-secret
+  ccpimageprefix: registry.connect.redhat.com/crunchydata
+  rootsecretname: my-demo-db-postgres-secret
+status:
+  message: Cluster has been initialized
+  state: pgcluster Initialized
+  ```
+Save this YAML file and reload it.
+
+Navigate to Operators > Installed Operators > Service Binding Operator and click on the 'Instances' tab
+
+Click on the Service Binding Request you created in the previous section and edit its YAML file
+
+We will edit the `backingServiceSelector` . This section is used to find the backing service - our operator-backed DB instance called `db-demo`.
+
+Replace the `backingServiceSelector` block with th efollowing YAML snippet:
+```yaml
+  backingServiceSelectors:
+    - group: crunchydata.com
+      kind: Pgcluster
+      namespace: service-binding-demo
+      resourceRef: my-demo-db
+      version: v1
+```
+Complete YAML file once updated:
+```yaml
+apiVersion: apps.openshift.io/v1alpha1
+kind: ServiceBindingRequest
+metadata:
+  selfLink: >-
+    /apis/apps.openshift.io/v1alpha1/namespaces/readmeapp/servicebindingrequests/mysboproj-servicebindingrequest-example-servicebindingrequest
+  resourceVersion: '114116646'
+  name: mysboproj-servicebindingrequest-example-servicebindingrequest
+  uid: 20983d16-59f2-4b9b-ae56-e3371480821f
+  creationTimestamp: '2020-09-03T20:47:16Z'
+  generation: 2
+  namespace: readmeapp
+  ownerReferences:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: mysboproj
+      uid: e3ca87f2-0208-4b00-82bb-db4345f834dd
+  finalizers:
+    - finalizer.servicebindingrequest.openshift.io
+spec:
```

Comment:
There were some API changes that got introduced with https://github.com/redhat-developer/service-binding-operator/pull/608 . for eg, applicationselector to the application, backingServiceSelector to selectors. @ajm01 can you PTAL. :)

Reply:
@Avni-Sharma Would that be a validation of `master` or are the changes available on operatorhub ?

Reply:
Validation of master and `beta` channel where we have the changes in `0.2.0-387` on operator-hub

Reply:
Also , all the examples/ in the repo have the new APIs now